tensorflow/python/platform/BUILD

```diff
@@ -117,6 +117,7 @@ tf_py_test(
     tags = [
         "no_pip",
         "no_windows",
+	"no_oss",  # TODO(b/259319686) : Disable the tes
```

Comment:
why not use no_rocm?

Reply:
is this a cherry-pick from upstream?

Reply:
this is basically a cherry-pick from https://github.com/tensorflow/tensorflow/commit/695aaf9d9905bb2abb955e68525598af0984d10c
which sets it as no_oss on the 2.11 branch for the same reason.